.github/workflows/test.yml

```diff
@@ -14,16 +17,96 @@ on:
       - '.github/workflows/test.yml'
 
 env:
+  CARGO_INCREMENTAL: '1'
+  NETWORK: mainnet
   PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
-  ZONE: europe-west1-b
-  MACHINE_TYPE: n2-standard-8
-  DEPLOY_SA: cos-vm@${{ secrets.GCP_PROJECT_ID }}.iam.gserviceaccount.com
+  GAR_BASE: us-docker.pkg.dev/${{ secrets.GCP_PROJECT_ID }}/zebra
+  GCR_BASE: gcr.io/${{ secrets.GCP_PROJECT_ID }}
+  REGION: us-central1
+  ZONE: us-central1-a
+  MACHINE_TYPE: c2-standard-16
```

Comment:
ooooo 💻 💻 💻

Reply:
Oh, yeah. I'll reduce that. There's no major impact between 4 and 16 CPU, the only impact is in the first 5-15 min of heavy computation, which creates a difference of around ~10 min overall. 

I'm not sure how much impact we're _expecting_ from having a bunch of CPUs but is not much, to be honest.